flink-ml-lib/src/main/java/org/apache/flink/ml/classification/linear/LogisticGradient.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.classification.linear;
+
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.apache.flink.ml.common.linalg.BLAS;
+
+import java.io.Serializable;
+
+/** Utility class to compute gradient and loss for logistic loss. */
```

Comment:
Spark has pretty detailed comments for `LogisticGradient`. It also provides reference link to read detailed mathematical derivation. Do we also need to provide more information here?

Reply:
I am not sure whether we should do that detailed explanation. As the information is just on the website.

I just added one url there. What do you think?

Reply:
The link provided by Spark links to slides, which is not very easy to glance through.

It would be sufficient if we can have a link to wikipedia with the necessary mathematical formulas.

I am OK to just keep the slideshare link for now.